### ISSUE TITLE
pre-trained policy?

### ISSUE THREAD

**Issue by ronuchit** [AdaCompNUS/diner-dash-simulator#1]
Hi,

Thank you for creating this simulator! I think it will be a very useful and complex benchmark. I was wondering if the authors have trained any policies that work well in this environment, e.g. Pytorch or Tensorflow models mapping from states to actions? Or, do you have any heuristic policy implemented that tends to accrue high reward? I am asking because I would like to build a dataset of "good behavior" for use in an imitation learning framework.

Thank you,
Rohan

**Comment by siwei0729** [comment]
Hi,

Thanks for the question. We have added the heuristic policy. You can check [here](https://github.com/AdaCompNUS/diner-dash-simulator#to-get-heuristic-policy).
